README.md

```diff
@@ -1,2 +1,67 @@
-# Soul4ReactAdmin
-Demonstrator for using Soul database wrapper with ReactAdmin
+# Soul for React Admin
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+## Running the soul wrapper
+
+1. Copy the `chinook.db` database file to the core folder
+
+   ```
+    cp chinook.db /soul/core
```

Comment:
is there really a leading slash at the start of `/soul`?

Reply:
My bad, you don't need it, i will update the Readme